pkg/systemd/logs.js

```diff
@@ -109,10 +111,15 @@ $(function() {
             console.warn(cockpit.message(error));
         }
 
-        function prepend_entries(entries) {
-            for (var i = 0; i < entries.length; i++)
+        function prepend_entries(entries, following) {
```

Comment:
"following" is a weird name for this, it sounds like `tail -f` or similar. It would better be called `track_services` or so (or maybe I misunderstood what it's doing). But this function is only ever called with `true`, so please drop the condition/argument and just always track the services.